extensions/wikia/AdEngine/crowdin.conf

```diff
@@ -0,0 +1,4 @@
+[crowdin.project]
+projectid = "ext-AdEngine"
```

Comment:
@Grunny is it easy enough to "extract" this module to a separate extension in future? Does it adds additional work for our translators or it's fine if at one point in the future we decide to extract it and create new project and change the files' structure in `Wikia/app`? What do you think?